NomaiVR/ModConfig/ModConfig.cs

```diff
@@ -9,6 +9,7 @@ public class ModConfig
         public bool debugMode = true;
         public bool preventCursorLock = true;
         public bool showHelmet = true;
+        public bool disableHudInConversations = false;
```

Comment:
Can we use "hide" instead of "disable" here? Think it makes more sense since it doesn't disable behavior, just hides the visuals.

Also wondering if it should default to true or false 🤔 False keeps it closer to original game, but it really is annoying in VR to have the HUD on during dialogue. But then you might also miss the notifications saying something has been added to the ship log (which happens often during conversations). Thoughts?

Reply:
> hide 

Absolutely, no objections

About the default, my thought process was pretty similar to yours. I used `false` mostly to keep behaviour identical to the current version. If it were at all possible to keep notifications visible but hide the rest of the HUD, that would probably be the ideal solution. In that case it might not even be necessary to have a config setting, i can't think of any situation where the HUD would be useful while in a conversation.

I will look into the possibility of hiding just the specific HUD elements.

Reply:
It definitely is possible. But to know what you need to hide and what to keep, you might need to know what the HUD objects look like exactly (which usually requires looking at the scene in Unity). In the past I have shifted the position of specifically only the notifications in HUD, which means at some point I knew how to select only that part. Don't remember if that code is still there, you'll have to check in the HUD-related modules. In any case, if it isn't something immediately obvious, the solution in this PR is good enough, keeping the default to false.

And I agree that if the notifications remain visible, we don't even need an option (I'm always in favor of removing unecessary configurable options)